### PR TITLE
Prettify the poller + disco output

### PR DESCRIPTION
**Pull request by laf**
Output now looks like:

```bash
#### Load poller module storage ####
Storage /: 3% [RRD Disabled]
Storage /sys/fs/cgroup: 0% [RRD Disabled]
Storage /sys/fs/cgroup/cgmanager: 0% [RRD Disabled]
Storage /dev/shm: 0% [RRD Disabled]
Storage /run: 9% [RRD Disabled]
Storage /run/user/0: 0% [RRD Disabled]
Storage /run/user/997: 0% [RRD Disabled]

>> Runtime for poller module 'storage': 0.450 seconds
#### Unload poller module storage ####
```

Tidies things up a little.

Also fixed a bug where the device group include was setting debug.

### CODE REVIEW

**Comment by paulgear** [comment]
+1000 - why didn't we do this months/years ago? :-)

**Comment by murrant** [comment]
Nice!

I have a feeling I left the debug in there :-/

File: includes/device-groups.inc.php [hunk 1]
```diff
@@ -237,7 +237,6 @@ function RunGroupMacros($rule,$x=1) {
 function UpdateGroupsForDevice($device_id)
 {
     global $debug;
```
Comment (murrant) [inline includes/device-groups.inc.php]: Remove the global too as it isn't used.

**Comment by laf** [comment]
Done

File: includes/discovery/functions.inc.php [hunk 2]
```diff
@@ -851,3 +848,13 @@ function avtech_add_sensor($device, $sensor) {
     return true;
 }
 
+function load_discovery_module($module, $device, $attribs) {
+    global $config, $valid;
+    $module_start = microtime(true);
+    echo "#### Load disco module $module ####\n";
+    include "includes/discovery/$module.inc.php";
+    $module_time = microtime(true) - $module_start;
+    $module_time = substr($module_time, 0, 5)
```
Comment (murrant) [inline includes/discovery/functions.inc.php]: Missing semi-colin

File: includes/polling/functions.inc.php [hunk 3]
```diff
@@ -237,17 +237,14 @@ function poll_device($device, $options) {
         if ($options['m']) {
             foreach (explode(',', $options['m']) as $module) {
                 if (is_file('includes/polling/'.$module.'.inc.php')) {
-                    include 'includes/polling/'.$module.'.inc.php';
+                    load_poller_module($module, $device, $attribs);
```
Comment (murrant) [inline includes/polling/functions.inc.php]: The call to load_poller_module() has too many arguments starting with $attribs.

File: includes/polling/functions.inc.php [hunk 4]
```diff
@@ -237,17 +237,14 @@ function poll_device($device, $options) {
...
             }
         }
         else {
             foreach ($config['poller_modules'] as $module => $module_status) {
                 if ($attribs['poll_'.$module] || ( $module_status && !isset($attribs['poll_'.$module]))) {
-                    $module_start = microtime(true);
-                    include 'includes/polling/'.$module.'.inc.php';
-                    $module_time = microtime(true) - $module_start;
-                    echo "Runtime for polling module '$module': $module_time\n";
+                    load_poller_module($module, $device, $attribs);
```
Comment (murrant) [inline includes/polling/functions.inc.php]: The call to load_poller_module() has too many arguments starting with $attribs.

Comment (f0o) [inline includes/polling/functions.inc.php]: how do you mean?

Comment (murrant) [inline includes/polling/functions.inc.php]: This is an error from scrutinizer-ci.  The load_poller_module() only has two arguments, but three are given in these two calls.

**Comment by laf** [comment]
Should be sorted now. Thanks.

**Pull request merged**